### DISCUSSION TITLE
Secondary Sidebar (or markdown doc) in a seperate window.

### DISCUSSION THREAD

**Discussion by alan-sims** [posit-dev/positron#4402] [Q&A]
Hello, 

I was wondering if there is a way to have the secondary sidebar in a separate window?
I do a lot of editing of Rmarkdown documents and like that Rstudio has the html file pop out in a new window so I can keep the main editor nice and wide. Is there a way to do this with Positron?

**Comment by juliasilge** [comment]
There is not currently a way to pop out the whole secondary sidebar, no, but you _can_ pop out the file in the Viewer pane. Click on this button:

![pop-out-html](https://github.com/user-attachments/assets/36bf0e08-2cae-42bf-9653-fcfef1ddc07a)

That will pop out the HTML file viewer into a browser window and you can then make your source editor wide.

**Reply by alan-sims** [reply]
Thank you for your response. This somewhat solves the problem. However, every time I re-render my markdown doc, it now updates in the browser window but the secondary sidebar also comes back with the preview, meaning I need to click it each time to make it go away and keep the browser window.
